### PR TITLE
Standardize PH events

### PR DESCRIPTION
**Pull request by joshenlim**
As per PR title
- Add TS enum for `TelemetryActions`, avoid using arbitrary strings so that we can easily reference what events we're tracking through the dashboard
- Simplified `sendEvent` mutation to just accept `action` and `properties`
- Removed unnecessary event sending when signing up, or signing in via MFA

### CODE REVIEW

**Review by alaister — APPROVED** [review]
LGTM!

- Tested on preview by verifying various events are being sent correctly

**Comment by joshenlim** [comment]
i'll wait for @4L3k51 to give the green light before merging this 🙂 🙏

**Pull request merged**
